tao_compiler/mlir/disc/tests/regression/corner_case.cc

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "tensorflow/compiler/mlir/disc/tests/mlir_feature_test.h"
+#include "tensorflow/compiler/mlir/disc/tests/mlir_test.h"
+#include "tensorflow/core/platform/test.h"
+
+namespace mlir_test {
+
+const std::string c_ft_path =
+    "tensorflow/compiler/mlir/disc/tests/regression/data/";
+
+#if CUDA_VERSION >= 11100
```

Comment:
Do we have this variable in in cpu-only env?

Reply:
Then this UT will not execute because an undefined macro is replaced with 0.

Reply:
Should we provide a proper ptxas for users with 11.0?

Reply:
> 

It is not the responsibility of the compiler itself. Even we provide a docker, we cannot prevent users to use other environments. (I do not think there will be that many users use our provided dockers by default.) Actually, the XLA engine will print warnings to complain about ptxas-11.0 if logging is enabled.